examples/example-wasm/src/hooks/useTaskManager.ts

```diff
@@ -0,0 +1,49 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { Task } from '../models/Task';
+
+const { useQuery, useRealm, useUser } = await import('@realm/react');
+
+/**
+ * Manages changes to the tasks in the realm.
+ */
+export function useTaskManager() {
+  const realm = useRealm();
+  const user = useUser();
+  const [requeryFlag, setRequeryFlag] = useState(false); // Temporary flag
+  const tasks = useQuery(Task, (collection) => collection, [requeryFlag]);
+
+  useEffect(() => {
+    // Temporary solution for making `useQuery` update the `tasks` reference.
+    // (The value doesn't matter, only that it is different from the initial value.)
+    setRequeryFlag(true);
+  }, []);
+
+  const addTask = useCallback((description: string) => {
+    console.log('Adding task:', description);
+    realm.write(() => {
+      realm.create(Task, { description, userId: user.id } as Task);
+    });
+  }, [realm, user.id]);
+
+  const toggleTaskStatus = useCallback((task: Task) => {
+    console.log('Toggling task status to:', task.isComplete ? 'Not done' : 'Done' );
```

Comment:
I'm wondering if we want to remove these logs?  I can see their value in debugging, but if someone is using the example, they would clearly see that the UI is responding to their actions.

Reply:
Can go ahead and remove the debugging logs 👍